streams/ru.m3u

```diff
@@ -47,13 +47,13 @@ https://hls-03-video.webcaramba.com/rutv/live.m3u8
 http://serv30.vintera.tv:8081/shoppinglive/shoppinglive_stream/playlist.m3u8
 #EXTINF:-1 tvg-id="Sochi24.ru",Sochi 24 HD
 http://serv30.vintera.tv:8081/sochi/sochi24_tv/playlist.m3u8
-#EXTINF:-1 tvg-id="",Sochi Live HD (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="SochiLiveHD.ru",Sochi Live HD (720p) [Not 24/7]
 http://serv30.vintera.tv:8081/sochi/sochi_stream/playlist.m3u8
 #EXTINF:-1 tvg-id="StarFamily.ru",Star Family
 http://89.254.128.5:22280/star_family_hd/video.m3u8
 #EXTINF:-1 tvg-id="",TRK 555 (720p)
 http://trk555.tv:8888/live
-#EXTINF:-1 tvg-id="",TVOE TV HD
+#EXTINF:-1 tvg-id="TvoyeTVYumor.ru",TVOE TV HD
```

Comment:
bro you can make it I believe

Reply:
I know why, the original TVoe isn't in the database, lol.

Reply:
The correct TVOE Humor btw : https://tvoetv.space/tvoetv_humor/video.m3u8